config/betterquesting/DefaultQuests.json

```diff
@@ -90,7 +90,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "It is probably impossible to try to play GregTech without using JEI. So here are some tips on how to use JEI to its full potential.\n\nThis pack will assume that the player\u0027s IQ is above 100. If you are confused, press §6R §ron an item in the questbook to see the item\u0027s crafting recipe, or §6U §rto see its uses.\n\n§eBookmark items regularly§r (default key §aA§r) so you don\u0027t have to type the names over and over.\n\nMost of the items in this pack have §emultiple recipes§r! Don\u0027t stop at the first recipe and think that item is impossible.\nSometimes a material is only available from §etiny dusts§r, sometimes it\u0027s available from a §efluid solution§r. It takes some practice. Oil processing and nuclear chemistry are pretty bad about this.\n\nYou can §epress @§r to see only items from that mod. For example, if you search for §c@gregtech§r, you will only see items from GregTech.\n\nYou can §epress $ §rto see only items from a specific entry in the Ore Dictionary. For example, if you search for §c$logWood§r, you will only see wooden logs.\n\nYou can §epress U §ron an item to see its uses.",
+          "desc:8": "It is probably impossible to try to play GregTech without using JEI. So here are some tips on how to use JEI to its full potential.\n\nFirst of all, you can press §6R §ron an item in the questbook and in JEI to see the item\u0027s crafting recipe, or §6U §rto see its uses.\n\n§eBookmark items regularly§r (default key §aA§r) so you don\u0027t have to type the names over and over.\n\nMost of the items in this pack have §emultiple recipes§r! Don\u0027t stop at the first recipe and think that item is impossible.\nSometimes a material is only available from §etiny dusts§r, sometimes it\u0027s available from a §efluid solution§r. It takes some practice. Oil processing and nuclear chemistry are pretty bad about this, §6§6ESPECIALLY§f in this pack.\n\n\nYou can §epress @§r to see only items from that mod. For example, if you search for §c@gregtech§r, you will only see items from GregTech.\n\nYou can §epress $ §rto see only items from a specific entry in the Ore Dictionary. For example, if you search for §c$logWood§r, you will only see wooden logs. This feature will be really useful for certain recipies!",
```

Comment:
nah

Reply:
can I at least remove 100 IQ comment it's stupid

Reply:
no its funny

Reply:
true